src/americana.js

```diff
@@ -45,9 +45,11 @@ function buildLayers() {
     lyrBoundary.stateCasing,
     lyrBoundary.countryCasing,
 
-    lyrWater.water,
     lyrWater.waterway,
     lyrWater.waterwayIntermittent,
+    lyrWater.waterLine,
```

Comment:
The waterline layer needs to draw under the water fill layer; otherwise, there would be visible seams at tile boundaries over water and at the mouths of rivers and harbors.

Reply:
Although there isn’t a seam at mouths of rivers, @zekefarwell pointed out that the waterline ends abruptly at the point where the `water=river` begins, seen here at zoom level 7.99 (but not at z7 or z8):

[<img src="https://user-images.githubusercontent.com/1231218/206877068-f02e4e00-3f83-42c8-9881-31ba641315a4.png" width="400" alt="St. Lawrence">](https://zelonewolf.github.io/openstreetmap-americana/#map=7.99/46.999/-70.696)

My initial inclination was not to worry much about it. Paper maps are unapologetic about beginning and ending strokes where they stop being applicable.

I tried giving lakes and rivers a fainter waterline at low zoom levels, but any color that’s discernible from the water fill ends up overemphasizing the water feature:

[<img src="https://user-images.githubusercontent.com/1231218/206877474-5e1fb447-e883-43ca-9826-8796cbb9f60d.png" width="400" alt="Dallas no waterline">](https://zelonewolf.github.io/openstreetmap-americana/#map=6/32.777/-96.655) <img src="https://user-images.githubusercontent.com/1231218/206877477-9fc3f025-3c76-44e3-9b93-58c4b944e389.png" width="400" alt="Dallas waterline">

Reply:
I tried out some variations for inland water as well.  It's definitely a challenge.  Between z6 and z8, there are many visible water bodies with quite highly detailed geometry.  The waterline can look ok on smoother water bodies, but on complex shorelines (reservoirs for example) the line ends zig-zagging back and for so much that it ends up looking thicker and/or darker.  I found it can look quite good below z6 as the water bodies become much more generalized at that point and many smaller ones are no longer visible.  I'm optimistic we can find a way to make a water line of some sort work on inland water bodies, but I'm happy to leave that for a future PR.